src/main/resources/config/application-clova.yml

```
@@ -0,0 +1,5 @@
+clova:
+  api-key:
+    ${CLOVA_API_KEY}
+  speech-url:
+      ${CLOVA_SPEECH_URL}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)